dask_sql/integrations/ipython.py

```diff
@@ -84,16 +76,46 @@ def sql(line, cell=None):
         if auto_include:
             dataframes = c._get_tables_from_stack()
 
-        return c.sql(cell, return_futures=False, dataframes=dataframes)
+        t0 = time.time()
+        res = c.sql(cell, return_futures=False, dataframes=dataframes)
+        if (
+            "CREATE OR REPLACE TABLE" in cell
+            or "CREATE OR REPLACE VIEW" in cell
+        ):
+            table = cell.split("CREATE OR REPLACE")[1]
+            table = table.replace("TABLE", "").replace("VIEW", "").split()[0].strip()
+            res = c.sql(f"SELECT * FROM {table}").tail()
+        elif (
+            "CREATE TABLE" in cell
+            or "CREATE VIEW" in cell
+        ):
+            table = cell.split("CREATE")[1]
+            table = table.replace("TABLE", "").replace("VIEW", "").split()[0].strip()
+            res = c.sql(f"SELECT * FROM {table}").tail()
+        print(f"Execution time: {time.time() - t0:.2f}s")
+        return res
```

Comment:
Models some code written by @randerzander : 
```
import time
import warnings

from IPython.core.magic import needs_local_scope, register_cell_magic

warnings.filterwarnings("ignore")


@register_cell_magic
@needs_local_scope
def sql(line, cell, local_ns):
    sql_statement = cell.format(**local_ns)
    t0 = time.time()
    res = c.sql(sql_statement)
    if (
        "CREATE OR REPLACE TABLE" in sql_statement
        or "CREATE OR REPLACE VIEW" in sql_statement
    ):
        table = sql_statement.split("CREATE OR REPLACE")[1]
        table = table.replace("TABLE", "").replace("VIEW", "").split()[0].strip()
        res = c.sql(f"SELECT * FROM {table}").tail()
    elif "CREATE OR REPLACE MODEL" in sql_statement:
        res = c.sql(sql_statement)
    else:
        res = res.compute()
        # since we want to pass DF results to Plotly, anytime we pull query results to the client
        # we go ahead and convert any cuDF DF to Pandas so Plotly can graph it
        #if GPU:
        #    res = res.to_pandas()
    print(f"Execution time: {time.time() - t0:.2f}s")
    return res


# avoid name conflicts for automagic to work on line magics.
del sql
```

Reply:
This reminds me.. will this magic work with variable substitution by name?

@ayushdg pointed me to the `@needs_local_scope` decorator. For example, if you have a variable `val=1234` defined, then the cell:

```
%%sql

select {val} as example from my_table
```

would return `1234` in the `example` column.

Reply:
Oh, cool! I'll re-add that part now.